src/wp-includes/user.php

```diff
@@ -34,6 +34,9 @@ function wp_signon( $credentials = array(), $secure_cookie = '' ) {
 	if ( empty( $credentials ) ) {
 		$credentials = array(); // Back-compat for plugins passing an empty string.
 
+		$credentials['user_login']    = '';
+		$credentials['user_password'] = '';
+
```

Comment:
Curious, why not initialize `$credentials` with these new values in a single statement?

Reply:
Yeah, that's what I ended up doing here in [r55301](https://core.trac.wordpress.org/changeset/55301) 👍